include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -254,17 +268,32 @@ struct TargetReduceLoc
 {
   TargetReduceLoc() = delete;
   TargetReduceLoc(const TargetReduceLoc &) = default;
-  explicit TargetReduceLoc(T init_val, IndexType init_loc)
+  explicit TargetReduceLoc(T init_val_, IndexType init_loc,
+                           T identity_ = Reducer::identity)
       : info(),
-        val(Reducer::identity, Reducer::identity, info),
+        val(identity_, identity_, info),
         loc(init_loc, IndexType(RAJA::reduce::detail::DefaultLoc<IndexType>().value()), info),
-        initVal(init_val),
-        finalVal(Reducer::identity),
+        initVal(init_val_),
+        finalVal(identity_),
         initLoc(init_loc),
         finalLoc(IndexType(RAJA::reduce::detail::DefaultLoc<IndexType>().value()))
   {
   }
 
+  void reset(T init_val_,
+             IndexType init_local_ =
+             IndexType(RAJA::reduce::detail::DefaultLoc<IndexType>().value()),
+             T identity_ = Reducer::identity)
+  {
+    operator T();
+    val.reset(identity_);
+    loc.reset(init_local_);
```

Comment:
@MrBurmark I took another pass at the reset method for _Loc variants. This is now consistent with the new changes with the other backends. The only thing I'm not 100% sure on is my usage on Reducer::identity here.

Reply:
Identity is the identity value for the Reducer Operator which only acts on the val so this looks right, just needs ().

Reply:
Thanks! Interestingly, in this case ``Reducer::identity`` is just a float so we can get away with not having (). This is the way it's used in the constructor. Which actually confuses me since its not consistent with the other version in this file.

Reply:
My bad, I thought Reducer::identity was a type.